controller/Review.php

```diff
@@ -100,68 +104,88 @@ public function init(): void
 
         $params = $this->getPsrRequest()->getQueryParams();
 
-        if (isset($params['serviceCallId'])) {
-
-            /** @var DeliveryExecutionFinderService $finder */
-            $finder = $this->getServiceLocator()->get(DeliveryExecutionFinderService::SERVICE_ID);
-
-            $data = $dataBuilder->create()->build($params['serviceCallId'], $finder->getShowScoreOption($this->ltiSession->getLaunchData()));
+        try {
+            if (isset($params['serviceCallId'])) {
+                /** @var DeliveryExecutionFinderService $finder */
+                $finder = $this->getServiceLocator()->get(DeliveryExecutionFinderService::SERVICE_ID);
+                $this->checkPermissions($finder->findDeliveryExecution($this->ltiSession->getLaunchData()));
+                $data = $dataBuilder->create()->build(
+                    $params['serviceCallId'],
+                    $finder->getShowScoreOption($this->ltiSession->getLaunchData())
+                );
+            }
+            $this->returnJson($data ?? []);
+        } catch (\common_exception_ClientException $e) {
+            $this->returnJson([
+                'success' => false,
+                'type' => 'error',
+                'message' => $e->getUserMessage()
+            ]);
         }
-
-        $this->returnJson($data ?? []);
     }
 
     /**
      * Provides the definition data and the state for a particular item
      */
     public function getItem(): void
     {
-        $params = $this->getPsrRequest()->getQueryParams();
+        try {
+            $params = $this->getPsrRequest()->getQueryParams();
 
-        $deliveryExecutionId = $params['serviceCallId'];
-        $itemDefinition = $params['itemUri'];
+            $deliveryExecutionId = $params['serviceCallId'];
```

Comment:
can we run the phpcs -standard=PSR12 on this please (from ide will be better to just select the area changed)?

Reply:
`phpcbf ../ltiTestReview/controller/Review.php --extensions=php --standard=PSR12

No fixable errors were found

Time: 42ms; Memory: 8MB`